### PR TITLE
Navigation for Walk-Throughs

### PR DESCRIPTION
**Pull request by FFincannon**

### CODE REVIEW

File: content/OATH/.conf.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
 	"Importing_keys",
 	"Card_edit",
 	"SSH_authentication"
```
Comment (elukewalker) [inline content/OATH/.conf.json:5]: @FFincannon, a missing comma is causing the build to fail.

File: content/OTP/.conf.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "Libraries",
     "OTPs_Explained",
     "OTP_Walk-Through"
```
Comment (elukewalker) [inline content/OTP/.conf.json:6]: @FFincannon, this line of the json object is also missing a comma.

**Review by elukewalker — CHANGES_REQUESTED** [review]
Will merge after commas are added.

**Comment by FFincannon** [comment]
@elukewalker , issues resolved per request

**Comment by FFincannon** [comment]
@elukewalker : Added missing commas and resolved the conflict. This PR is now ready for merging.

**Review by elukewalker — APPROVED** [review]

**Pull request merged**